wsl2_gpu_guide/README.md

```diff
@@ -0,0 +1,70 @@
+# WSL2 gpu guide
```

Comment:
Can you change this to match up with the README? That is, the README should also say "WSL2 GPU Guide" (the directory name) and the short description should be the same as well. The [example](https://github.com/tensorflow/build/tree/master/directory_template) includes all that too -- the long description will fit under the section separator (`***`)